contrib/terraform/gcp/sample-inventory/cluster.tfvars

```diff
@@ -0,0 +1,66 @@
+###### Environment Details or ######
+env = "open-source" 
+
+######### user_name #
+user_name = "app"
+
+######## Region ##############
+region = "us-central1"
+
+######### Kube automation folder where you would like to clone the kubespray project #################
+kube_automation_folder ="/home/app/kube_automation"
+
+########### Kubespray repo URL hosted  in the github ####
+kubespray_repo_url = "https://USER_NAME:PASSWORD@github.com/kubernetes-sigs/kubespray.git"
```

Comment:
Default value should be without username and password since they are not required to clone kubernetes-sigs/kubespray.

Reply:
The cluster.tfvars does not have any value default username and password. 

USER_NAME & PASSWORD are just place holders.